docs/documentation/schemas.md

```diff
@@ -72,7 +74,7 @@ CSV transformer could be build with help of `CsvTransformer.CsvTransformerBuilde
 === "Kotlin"
 
     ``` kotlin
-        val transformer = CsvTransformer.<String>builder().header(true).separator(separator).build()
+        val transformer = CsvTransformer.builder<String>().header(true).separator(separator).build()
```

Comment:
I'm not a kotlin person, but in java on the right of an assignment you can usually just use a diamond operator `<>` without specifying the type because it's inferred by the JVM. (Though in that case it is needed on the left.)

Reply:
I thought the same, but it doesn't compile. But my change fixes the problem.

You can try by yourself:
``` kotlin
    val faker = BaseFaker()

    val schema = Schema.of(
        field("first_name",
            Supplier { faker.name().firstName() }),
        field("last_name",
            Supplier { faker.name().lastName() }),
        field<String, String>("address",
            Supplier { faker.address().streetAddress() })
    )

    val transformer = CsvTransformer.builder<String>().header(true).separator(" : ").build()

    val csv = transformer.generate(schema, 2)
    print(csv)
```

Reply:
Okay, thanks

Reply:
you're welcome!